akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala

```diff
@@ -350,6 +351,10 @@ object Replicator {
     def dataValue: A = data
   }
 
+  final case class Deleted[A <: ReplicatedData](key: Key[A]) extends NoSerializationVerificationNeeded {
```

Comment:
Add a new section for Distributed Data in akka-docs/rst/project/migration-guide-2.4.x-2.5.x.rst and mention this change.